UPGRADE.md

```diff
@@ -6,6 +6,10 @@ awareness about deprecated code.
 - Use of our low-overhead runtime deprecation API, details:
   https://github.com/doctrine/deprecations/
 
+# Upgrade to 1.8
+
+Executor and Purger classes are now final. You should not extend them.
```

Comment:
You made the `AbstractExecutor` internal, should that be mentioned as well? What's the consequence of marking a class as `internal` for the users? I think this means that:
- they should not extend it
- they cannot even type hint against it in their code

Reply:
Exact. I updated the UPGRADE doc.

Reply:
Great! Now you just have to push it to github :stuck_out_tongue:

Reply:
🤦